accessibility_monitoring_platform/apps/cases/views.py

```diff
@@ -201,19 +198,22 @@ class CaseDetailView(DetailView):
     View of details of a single case
     """
 
-    model: Type[Case] = Case
+    model: type[Case] = Case
     context_object_name: str = "case"
 
-    def get_context_data(self, **kwargs) -> Dict[str, Any]:
-        """Add undeleted contacts to context"""
-        context: Dict[str, Any] = super().get_context_data(**kwargs)
-        case: Case = self.object
+    def get_context_data(self, **kwargs) -> dict[str, Any]:
+        """Add case detail sxections to context"""
```

Comment:
Mispelling of sections

Reply:
Fixed. 4dd34c1f899d7a682f50c635b2ca227bdafe18a1